src/container/HashContainer/Base/index.ts

```diff
@@ -1,11 +1,19 @@
 import { Base, Container } from '@/container/ContainerBase/index';
 
+export const enum HashContainerConst {
```

Comment:
See `HashSet.test.ts` and `HashMap.test.ts`, I think you may need to make modifications to the constants?

Just change `HashContainer.xxx` to `HashContainerConst.xxx`.

Reply:
HashContainerConst is a real constant, so it is impossible to force modification

Reply:
Is it difficult to fill 100% coverage without changing that constant?
If it is, I'll find an alternative method

Reply:
MayBe you can change the hash function.

Like this: `new HashSet([], undefined, (x) => Number(x))`.

But I haven't tried.

Change the constant is an easy way to get high coverage.

Reply:
Thank you. The only problem left now is the `HashMap` and `HashSet` test. I'm trying to fill the coverage without changing the constant.